backend/app/groups/service.py

```
@@ -1,11 +1,13 @@
 from fastapi import HTTPException, status
 from app.database import get_database
-from bson import ObjectId
+from app.config import logger
+from bson import ObjectId, errors
 from datetime import datetime, timezone
 from typing import Optional, Dict, Any, List
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)